advanced/is-pattern-2/README.md

```diff
@@ -0,0 +1,281 @@
+  Docker image which packages WSO2 Updates, available at [WSO2 Private Docker Registry](https://docker.wso2.com/). In order
+  to use these images, you need an active [WSO2 Subscription](https://wso2.com/subscription).
+  <br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm 3](https://helm.sh/docs/intro/install/)
+  and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the steps provided in the
+  following quick start guide.<br><br>
+
+* An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup).<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/).<br><br>
+
+* Add the WSO2 Helm chart repository.
+
+    ```
+     helm repo add wso2 https://helm.wso2.com && helm repo update
+    ```
+
+## Quick Start Guide
+
+### 1. Install the Helm Chart
+
+You can install the relevant Helm chart either from [WSO2 Helm Chart Repository](https://hub.helm.sh/charts/wso2) or by source.
+
+**Note:**
+
+* `NAMESPACE` should be the Kubernetes Namespace in which the resources are deployed.
+
+#### Install Chart From Source
+
+>In the context of this document, <br>
+>* `KUBERNETES_HOME` will refer to a local copy of the [`wso2/kubernetes-is`](https://github.com/wso2/kubernetes-is/)
+Git repository. <br>
+>* `HELM_HOME` will refer to `<KUBERNETES_HOME>/advanced`. <br>
+
+##### Clone the Helm Resources for WSO2 Identity Server Git repository.
+
+```
+git clone https://github.com/wso2/kubernetes-is.git
+```
+
+##### Deploy Helm chart for a clustered deployment of WSO2 Identity Server.
+
+ **Helm version 3**
+
+ - Create the Kubernetes Namespace to which you desire to deploy the Kubernetes resources.
+ 
+    ```
+    kubectl create ns <NAMESPACE>
+    ```
+
+ - Deploy the Kubernetes resources using the Helm Chart
+ 
+    ```
+    helm install <RELEASE_NAME> <HELM_HOME>/is-pattern-1 --version 5.10.0-2 --namespace <NAMESPACE> --dependency-update
+    ```
+
+The above steps will deploy the deployment pattern using WSO2 product Docker images available at DockerHub.
+
+If you are using WSO2 product Docker images available from WSO2 Private Docker Registry,
+please provide your WSO2 Subscription Credentials via input values (using `--set` argument). 
+
+Refer the following example.
+
+```
+ helm install --name <RELEASE_NAME> <HELM_HOME>/is-pattern-1 --version 5.10.0-2 --namespace <NAMESPACE> --set wso2.subscription.username=<SUBSCRIPTION_USERNAME> --set wso2.subscription.password=<SUBSCRIPTION_PASSWORD>
+```
+
+### 2. Obtain the external IP
+
+Obtain the external IP (`EXTERNAL-IP`) of the Identity Server Ingress resource, by listing down the Kubernetes Ingresses.
+
+```  
+kubectl get ing -n <NAMESPACE>
+```
+
+The output under the relevant column stands for the following.
+
+- NAME: Metadata name of the Kubernetes Ingress resource (defaults to `wso2is-pattern-1-identity-server-ingress`)
```

Comment:
@darsham3 it would be better to add the Dashboard hostname details and instructions associated with it as well. Please sync this with the `NOTES.txt` output.